openquake/commands/mosaic.py

```diff
@@ -143,7 +147,9 @@ def run_site(lonlat_or_fname, *, hc: int = None, slowest: int = None,
         from_file(lonlat_or_fname, concurrent_jobs)
         return
     lon, lat = lonlat_or_fname.split(',')
-    params = get_params_from(dict(lon=lon, lat=lat, vs30=vs30))
+    params = get_params_from(
+        dict(lon=lon, lat=lat, vs30=vs30), config.directory.mosaic_dir,
```

Comment:
Not here, I want to be able to use `oq mosaic run_site` also in the USA for non-AELO calculations